vendor/github.com/aws/aws-sdk-go/service/s3/api.go

```diff
@@ -31314,8 +31314,14 @@ const (
 	// BucketCannedACLPublicReadWrite is a BucketCannedACL enum value
 	BucketCannedACLPublicReadWrite = "public-read-write"
 
+	// BucketCannedACLAWSExecRead is a BucketCannedACL enum value
+	BucketCannedACLAWSExecRead = "aws-exec-read"
+
 	// BucketCannedACLAuthenticatedRead is a BucketCannedACL enum value
 	BucketCannedACLAuthenticatedRead = "authenticated-read"
+
+	// BucketCannedACLLogDeliveryWrite is a BucketCannedACL enum value
+	BucketCannedACLLogDeliveryWrite = "log-delivery-write"
```

Comment:
Unfortunately, we can't make changes in the vendored dependencies, because they'll be wiped out the next time we update the AWS SDK. When we run into missing constants, the first thing to do is double-check that they haven't been added in a new version of the SDK. If they haven't, then we can define our own constants in the directory `aws/internal/service/s3` and also file an issue with AWS either on the SDK or with support